lib/msgpack/core_ext.rb

```diff
@@ -126,6 +134,24 @@ def to_msgpack_with_packer(packer)
   end
 end
 
+class Time
+  include MessagePack::CoreExt
+
+  def self.from_msgpack_ext(payload)
```

Comment:
This may break existing user code if users defined their own `Time.from_msgpack_ext` (and `Time#to_msgpack_ext`).
It seems better not to have these methods, and to add options to enable predefined types on Factory class. What do you think about this idea?
(It requires patches for C/Java ext, so I'm ok to do it by myself, out of this change)

Reply:
The latest code no longer break the existing code: Instead, use `MessagePack::Time::{Packer,Unpacker}` by hand:

```ruby
      factory.register_type(
        MessagePack::Timestamp::TYPE,
        Time,
        packer: MessagePack::Time::Packer,
        unpacker: MessagePack::Time::Unpacker
      )
```

as tests show.